coalib/misc/Shell.py

```diff
@@ -52,6 +54,11 @@ def run_interactive_shell_command(command, **kwargs):
     """
     if not kwargs.get('shell', False) and isinstance(command, str):
         command = shlex.split(command)
+    else:
+        command = list(command)
+
+    if sys.platform.startswith('win'):  # pragma: no cover
```

Comment:
Thanks! Didn't know about `platform.system`. Really the better solution :)

**CC** @justuswilhelm